spec/system/sessions_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "gds_api/test_helpers/account_api"
+require "gds_api/test_helpers/email_alert_api"
+
+RSpec.describe "Sessions", type: :system do
+  include GdsApi::TestHelpers::AccountApi
+  include GdsApi::TestHelpers::EmailAlertApi
+  include GovukPersonalisation::TestHelpers::Features
+
+  before do
+    stub_content_store_has_item("/")
+  end
+
+  context "Given a signing in user" do
+    it "Logs the user in and redirect them to manage their account" do
+      given_a_successful_login_attempt
+      visit(new_govuk_session_callback_path(code: "code", state: "state"))
+
+      expect(page.current_url).to eq("#{GovukPersonalisation::Urls.one_login_your_services}/")
+    end
+
+    it "Logs the user in and send them to a redirect path if supplied" do
+      @redirect_path = "/email/subscriptions/account/confirm?frequency=immediately&return_to_url=true&topic_id=some-page-with-notifications"
+      given_a_successful_login_attempt
+
+      expect_to_be_redirected { when_i_return_from_digital_identity }
```

Comment:
Yeah, this is a bit of a crazy one. I don't fully understand why it needs this bizarre yield/rescue block or what that's supposed to be doing. If it's okay, let's come back to this in a future PR.

Reply:
I think it might be worth adding back in the original comment that's presumably been lost with the conversion? As it kind of makes more sense: https://github.com/alphagov/frontend/blob/94fe22def95df71cf6f89943a86c35d1e4fb7f76/test/integration/sessions_test.rb#L52-L58

Reply:
Okay, playing around with this a bit I think I've got a better plan (and I've commented it), which is to use the normal `expect(page.current_url).to eq (redirect path)`, and to make that route available by temporarily adding it pointing to the homepage.